packages/pds/tests/_util.ts

```diff
@@ -93,6 +93,7 @@ export const runTestServer = async (
     didCacheStaleTTL: HOUR,
     jwtSecret: 'jwt-secret',
     availableUserDomains: ['.test'],
+    rateLimitsEnabled: false,
```

Comment:
Since the rate limits are in the critical path, it might be nice to get at least one simple test going with rate limiting enabled, i.e. basic gutcheck to ensure they generally allow requests through.

Reply:
yup yup - that was totally on my TODOs & then forgot that i never got it done 😛